src/tests/ftest/control/dmg_pool_query_ranks.py

```diff
@@ -141,8 +142,21 @@ def test_pool_query_ranks_mgmt(self):
         self.log.info("Starting reintegrating ranks: all_ranks=%s", all_ranks)
         for rank in all_ranks:
             self.log.debug("Reintegrating rank %d", rank)
-            self.pool.reintegrate(rank)
 
+            counter = 0
+            cmd_succeed=False
+            while counter < 3:
+                try:
+                    result = self.pool.reintegrate(rank)
+                except CommandFailure:
```

Comment:
This comes from exception_utils so probably need

from exception_utils import CommandFailure